demisto_sdk/commands/setup_env/setup_environment.py

```diff
@@ -85,13 +90,31 @@ def get_docker_python_path(docker_prefix: str) -> List[str]:
             docker_python_path.append(
                 f"{docker_prefix}/{path.relative_to(CONTENT_PATH.absolute())}"
             )
+
+    if not COMMON_SERVER_PYTHON_PATH.exists():
+        try:
+            common_server_python_contnet = TextFile.read_from_github_api(
```

Comment:
Does this support the offline mode env var?

Reply:
nope, need to add, good catch

Reply:
will be done in https://github.com/demisto/demisto-sdk/pull/4098

Reply:
fixed in https://github.com/demisto/demisto-sdk/pull/4098